qoffer-domain/src/main/groovy/life/qbic/portal/portlet/offers/identifier/TomatoId.groovy

```diff
@@ -0,0 +1,47 @@
+package life.qbic.portal.portlet.offers.identifier
+
+import groovy.transform.CompileStatic
+
+/**
+ * <class short description - 1 Line!>
+ *
+ * <More detailed description - When to use, what it solves, etc.>
+ *
+ * @since <versiontag>
+ */
+@CompileStatic
```

Comment:
description missing

Reply:
```suggestion
/**
 * Represents and identifier for an offer
 *
 * Contains the random part, project conserved part and the version of an identifier
 *
 * @since 0.1.0
 */
@CompileStatic
```